data/json/items/generic.json

```
@@ -2665,6 +2665,30 @@
     "volume": "750 ml",
     "category": "spare_parts"
   },
+  {
+    "id": "gatling_mech_laser_salvaged",
+    "copy-from": "cerberus_laser",
+    "looks_like": "gatling_mech_laser",
+    "type": "GENERIC",
+    "color": "blue",
+    "name": { "str": "salvaged CMES gatling laser" },
+    "description": "A rapid-fire weapon system stripped from a wrecked combat exoskeleton.  No longer usable as a weapon in its current condition.",
+    "material": [ "superalloy", "plastic" ],
+    "weight": "39500 g",
```

```suggestion
    "weight": "40 kg",
```
no suspiciously specific values!

---

I guess we can but this was a straight copy of the existing weapon's value, the rebuilt version rounds up because you add material to it so. XD